code/z_adventurezones/void.dm

```diff
@@ -39,6 +39,12 @@ CONTENTS:
 		for(var/mob/living/carbon/human/H in src)
 			H.client?.playAmbience(src, AMBIENCE_FX_2, 50)
 
+/area/crunch/artifact_boh_pocket_dimension
```

Comment:
why crunch

Reply:
I wanted to reuse the void ambience mostly